src/bun.js/bindings/JSBuffer.cpp

```diff
@@ -743,9 +743,22 @@ static inline JSC::EncodedJSValue jsBufferConstructorFunction_concatBody(JSC::JS
     }
 
     if (callFrame->argumentCount() > 1) {
```

Comment:
We could simplify this a little and say something like:

```
JSValue lengthValue = callframe->argument(1)
if (lengthValue.isUndefined()) {
  // …
```

IIRC, argument() will return undefined if it’s outside the bounds of argumentCount()